app/models/application_record.rb

```diff
@@ -13,4 +13,10 @@ def generate_slug(hash_string)
   rescue ActiveRecord::RecordNotUnique
     Digest::SHA1.hexdigest("#{hash_string}#{SecureRandom.uuid}")[8..14] if hash_string
   end
+
+  protected
+
+  def set_slug_from_id
```

Comment:
We're actually going to be getting rid of slugs in a later ticket, per our discussion in Slack - I don't think you need to change this now but we're gonna end up pulling it out. @csprayregen can we prioritize the slug ticket soonish?

Reply:
just saw that.  Yup -- we can decide which way to go (uuids v. slugs) and go from there.